### PR TITLE
Revamping the UX/UI of the Launcher Screen

### LINKED ISSUE #276: Revamp the UX of the Launcher Activity
**Actual Behaviour.**  
The launcher activity has an outdated UX/UI

**Expected Behaviour**  
Revamp the UX of the Launcher Activity.

**Would you like to work on the issue?**
Yes would be working on this.

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #276 

**Changes**: Revamping the UX/UI of the Launcher Screen

**Screenshot/s for the changes**: 
Screenshot_20190607-205027
Screenshot_20190607-205721
Screenshot_20190607-205316

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-debug.zip](https://github.com/fossasia/neurolab-android/files/3266747/app-debug.zip)

### CODE REVIEW

**Comment by jddeep** [comment]
@CloudyPadmal The basic structure is ready. Need to make a few more design tweaks and updates overall. After that, we will be good to go.

**Comment by jddeep** [comment]
@adityastic I saw you are good at designing. I have got an interesting job for you :D Can you design specific program mode images/icons for 4 cards in the launcher. (See in the screenshot, have used the same image for every card)

**Comment by adityastic** [comment]
@jddeep don't worry, I'll take care of the icons. But change the design of the cards, try to shift the icon on the left side and the title and subtitle on the other. It will be more consistent

**Comment by adityastic** [comment]
I'll also design the background for the cards as well. Give me some time, I'll be back home in 3 hrs

**Comment by jddeep** [comment]
> @jddeep don't worry, I'll take care of the icons. But change the design of the cards, try to shift the icon on the left side and the title and subtitle on the other. It will be more consistent

Yeah i know. I have just got the structure ready. Need to make some final tweaks.

**Comment by CloudyPadmal** [comment]
> > @jddeep don't worry, I'll take care of the icons. But change the design of the cards, try to shift the icon on the left side and the title and subtitle on the other. It will be more consistent
> 
> Yeah i know. I have just got the structure ready. Need to make some final tweaks.

Make a PR when everything is ready.

File: app/src/main/res/layout/content_main.xml [hunk 1]
```diff
@@ -1,208 +1,223 @@
...
+            app:layout_constraintTop_toBottomOf="@+id/btn_focus">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:orientation="vertical">
+
+                <ImageView
+                    android:layout_width="match_parent"
+                    android:layout_height="80dp"
```
Comment (suresh-gandhi) [inline app/src/main/res/layout/content_main.xml]: remove the hardcoded value

File: app/src/main/res/layout/content_main.xml [hunk 2]
```diff
@@ -1,208 +1,223 @@
...
+                    android:src="@drawable/app_logo" />
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:padding="10dp"
```
Comment (suresh-gandhi) [inline app/src/main/res/layout/content_main.xml]: here also

File: app/src/main/res/layout/content_main.xml [hunk 3]
```diff
@@ -1,208 +1,223 @@
...
+                    android:text="Serial"
+                    android:textColor="@android:color/black"
+                    android:textSize="18sp" />
```
Comment (suresh-gandhi) [inline app/src/main/res/layout/content_main.xml]: here and everywhere else

**Review by suresh-gandhi — CHANGES_REQUESTED** [review]

**Comment by adityastic** [comment]
@jddeep change the status to WIP

**Review by adityastic — COMMENTED** [review]
@jddeep change the title to WIP

**Comment by jddeep** [comment]
WIP
launchfixui(1)

Will finalize once #280 get's done.

**Comment by adityastic** [comment]
@jddeep no, don't do this. Let me design the rest of the cards for you. Also the actionbar does not match the card theme

**Comment by jddeep** [comment]
> @jddeep no, don't do this. Let me design the rest of the cards for you. Also the actionbar does not match the card theme

@adityastic Relax. If you see I haven't made any commit xD. Will do once everything is finished. Just commented for an update as WIP.

**Comment by adityastic** [comment]
Oh xD great

**Comment by CloudyPadmal** [comment]
@adityastic why don't you send a direct PR with image resources? :thinking:

**Comment by adityastic** [comment]
@CloudyPadmal sounds better to me xD. @jddeep let me send a pr with the complete design change

**Comment by adityastic** [comment]
@CloudyPadmal sounds better to me xD

**Comment by jddeep** [comment]
> @adityastic why don't you send a direct PR with image resources? 🤔

Yup @CloudyPadmal that was basically even the #280 description xD

**Comment by jddeep** [comment]
@suresh-gandhi Peer review please :D

**Review by CloudyPadmal — APPROVED** [review]

**Review by mariobehling — APPROVED** [review]

**Comment by mariobehling** [comment]
Good job!

**Pull request merged**